infrastructure/metadata/infrastructure/server/webtransport-h3.sub.any.js.ini

```
@@ -0,0 +1,29 @@
+[webtransport-h3.sub.any.html]
+  [WebTransport server should be running and should handle a bidirectional stream]
+    expected:
+      if product == "safari": [FAIL, ERROR]
+      FAIL # TODO(bashi): Enable the WebTransport test server once aioquic dependency is resolved. Ditto below.
+
+[webtransport-h3.sub.any.window.html]
+  [WebTransport server should be running and should handle a bidirectional stream]
+    expected:
+      if product == "safari": [FAIL, ERROR]
+      FAIL
+
+[webtransport-h3.sub.any.worker.html]
+  [WebTransport server should be running and should handle a bidirectional stream]
+    expected:
+      if product == "safari": [FAIL, ERROR]
+      FAIL
+
+[webtransport-h3.sub.any.sharedworker.html]
+  [WebTransport server should be running and should handle a bidirectional stream]
+    expected:
+      if product == "safari": [FAIL, ERROR]
```

```suggestion
      if product == "safari": ERROR
```

It ought to always be ERROR in Safari for this test. Since it should be a harness error, I think you'll also have to move it out of the subtest section here. metadata/infrastructure/server/order-of-metas.any.js.ini is an example of what should probably look like.

---

Thanks! I used a pattern I found in title.any.js.ini in the same directory.